src/main/java/mate/academy/dao/UserDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.model.User;
+
+public interface UserDao {
+
+    Optional<User> findByLogin(String login);
```

Comment:
```suggestion
    Optional<User> findByEmail(String email);
```
We work with email, not login